packages/hawtio/src/plugins/camel/CamelContent.tsx

```diff
@@ -47,8 +48,8 @@ export const CamelContent: React.FunctionComponent = () => {
   interface NavItem {
     id: string
     title: string
-    component: React.FunctionComponent
-    isApplicable(node: MBeanNode): boolean
+    componentFn: () => JSX.Element
```

Comment:
Why change from Component to JSX.Element? Just curious, as both should work but I have usually seen the other approach. Did it make something easier?

Reply:
If it's because of 
```(
          <RouteDiagramContext.Provider value={ctx}>
            <RouteDiagram />
          </RouteDiagramContext.Provider>
 )```
 
 I believe you could just paste the part between parentheses and it should work. JSX.Element and React.FunctionComponent are interchangeable here. I would have to look deeper into it anyway to see if there are more differences